grails-app/conf/BootStrap.groovy

```diff
@@ -16,13 +16,18 @@ class BootStrap {
         if(skip){
             return;
         }
-
+        
         // since bootstrap loads expert trees and expert tree requires curator app on web2py, we cannot
         // proceed further as it will break the test on travis.
         if(Environment.current == Environment.TEST) {
             return ;
         }
 
+        //update alaDataresourceInfo with biocache urls
+        grailsApplication.config.alaDataresourceInfo.biocacheServiceUrl = grailsApplication.config.biocacheServiceUrl
```

Comment:
moving the config to a .properties file means the array based config values need to be set in a different way.